src/backend/core/migrations/0035_address_normalization.py

```
@@ -0,0 +1,175 @@
+
+from django.db import migrations, models
+import django.core.validators
+
+import core.mda.addresses
+
+ASCII_LOWER = str.maketrans(string.ascii_uppercase, string.ascii_lowercase)
+
+
+def _report(collisions, label):
+    if collisions:
+        print(
```

Ruff (0.16.2)

[warning] `print` found

Remove `print`

(T201)